packages/dendron-cli/src/commands/doctor.ts

```diff
@@ -66,6 +72,43 @@ export class DoctorCLICommand extends CLICommand<CommandOpts, CommandOutput> {
     });
   }
 
+  getWildLinkDestinations(notes: NoteProps[], wsRoot: string) {
+    let wildWikiLinks = [] as DLink[];
+    _.forEach(notes, (note) => {
+      const links = note.links;
+      if (_.isEmpty(links)) {
+        return;
+      }
+      wildWikiLinks = wildWikiLinks.concat(
+        _.filter(links, (link) => {
+          if (link.type !== "wiki") {
+            return false;
+          }
+          const destVault = link.to!.vault ? link.to!.vault : note.vault;
+          const noteExists = NoteUtils.getNoteByFnameV5({
+            fname: link.to!.fname as string,
+            vault: destVault,
+            notes: notes,
+            wsRoot: wsRoot,
+          }) as NoteProps;
```

Comment:
don't cast it here because it could be undefined. the !noteExists should still be valid without a cast